docs/_includes/subscribe.html

```diff
@@ -35,7 +35,19 @@ <h3 class="section-subheading text-muted">{{ site.data.sitetext.subscribe.text |
           </form>
 
         </div>
+
       </div>
+
+      <!--Mailing List-->
+      <div class="mailing-container">
+        <h1>Mailing Lists</h1>
+        <p>Engage in the Service Mesh Performance project. Join any of our mailinglists.</p>
+        <div class="mailing-btn-container">
```

Comment:
could you please put a gap between mailing and lists because they are two separate words.

Reply:
Sorry, Missed this one. I will fix it immediately.

Reply:
Fixed.